cmd/internal/find/parser/parse.go

```diff
@@ -9,7 +9,7 @@ import (
 type Result struct {
 	Paths     []string
 	Options   types.Options
-	Predicate types.EntryPredicate
+	Predicate *types.EntryPredicate
```

Comment:
It's not clear to me why this all changed to pointers.

Reply:
I was thinking b/c it's now a "struct" instead of a promoted function, it'd be a pointer so we can mutate it. But now I see that was a mistake on my part. Will update to change this to values.

Reply:
Oh wait, it's so I can return nil if nothing was parsed. That's why I made it a pointer.